.Rbuildignore

```diff
@@ -19,3 +21,5 @@
 
 # vignette cache
 ^vignettes/cached/.*\.Rmd$
+# large vignette excluded on CRAN
+^vignettes/plate_scoring_examples.Rmd$
```

Comment:
@klmr do you know if this still will be built for the pkgdown?

Reply:
Yes: see [pkgdown.r-lib.org/reference/build_articles.html](https://pkgdown.r-lib.org/reference/build_articles.html#suppressing-vignettes:~:text=If%20you%20want%20articles%20that%20are%20not%20vignettes%2C%20either%20put%20them%20in%20subdirectories%20or%20list%20in%20.Rbuildignore.).